cmd/agent/dist/conf.d/jmx.d/conf.yaml.example

```diff
@@ -20,11 +20,14 @@ instances:
   #   tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar # To be set when process_name_regex is set
 
   #   name: jmx_instance
-  #   # java_bin_path: /path/to/java # Optional, should be set if the agent cannot find your java executable
-  #   # java_options: "-Xmx200m -Xms50m" # Optional, Java JVM options
-  #   # trust_store_path: /path/to/trustStore.jks # Optional, should be set if ssl is enabled
-  #   # trust_store_password: password
-  #   # refresh_beans: 600 # Optional (in seconds), default is 600 seconds. Sets refresh period for refreshing matching MBeans list.
```

Comment:
the double `#` here were meaningful to reflect that the options are optional. Just pointing that out, not sure it should make sense to keep things this way

Reply:
Only `host` and `port` are really non-optional and we don't do this in most configuration so it looks inconsistent to me.